plugin/helpers.go

```diff
@@ -78,6 +80,10 @@ type Helpers interface {
 	//
 	// Minimum server version: 5.18
 	InstallPluginFromURL(downloadURL string, replace bool) (*model.Manifest, error)
+
+	// GetPluginAssetURL builds a URL to the given asset in the assets directory.
+	// Use this URL to link to assets from the webapp, or for third-party integrations with your plugin.
+	GetPluginAssetURL(pluginID, asset string) (*url.URL, error)
```

Comment:
@Lumexralph and I have both looked into how passing the `pluginID` can be avoided and both didn't fond a way `HelpersImpl` could get `pluginID`. Do you see a way?

Reply:
At present, I don't know of a good way to do this myself.